modules/placekey/redshift/sql/H3_ASPLACEKEY.sql

```diff
@@ -0,0 +1,16 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION @@RS_PREFIX@@placekey.H3_ASPLACEKEY
+(h3_index VARCHAR(15)) 
```

Comment:
This is just curiosity, I guess we are assuming that all the H3_index ara maximum 15 characters and placekey maximum 19 right?

Reply:
Yes, it's what placekey and h3 format says. A checking for free...